<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GfxPaint::TransformEditorWidget</class>
 <widget class="QWidget" name="GfxPaint::TransformEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>221</width>
    <height>155</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,1,1">
   <item row="0" column="0">
    <widget class="QLabel" name="modeLabel">
     <property name="text">
      <string>&amp;Mode</string>
     </property>
     <property name="buddy">
      <cstring>modeComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QTableView" name="transformTableView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::AllEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="identityButton">
     <property name="text">
      <string>&amp;Identity</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="modeComboBox">
     <item>
      <property name="text">
       <string>View</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Object</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Brush</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="independentCheckBox">
     <property name="text">
      <string>&amp;Independent</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>modeComboBox</tabstop>
  <tabstop>transformTableView</tabstop>
  <tabstop>identityButton</tabstop>
  <tabstop>independentCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
